common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1203,6 +1205,7 @@
             },
 
             hideClosedCaptions: function() {
+                var text = gettext('Turn on closed captioning');
```

Comment:
captioning or captions?

Reply:
It can be both I guess. Actually, I just copied this text from the code below in this function and placed it in a variable.